utils/hare-start

```diff
@@ -30,7 +30,37 @@ die() {
     exit 1
 }
 
+usage() {
+    cat <<EOF
+Usage: $PROG [<option>]...
+
+Start Hare and Motr services on local or all the nodes in the cluster.
+
+Options:
+  --node       Starts hare and motr services on current node only.
+  -h, --help   Show this help and exit.
```

Comment:
Wording: let's either use "Starts" and "shows" (in the meaning "It shows, it starts" - description of what something external to us would do) or "Start" and "show" (as if it is a command, not description). Let's not mix them.

Reply:
Yes, **Done**.